platforms/documentation/docs/src/snippets/java-platform/dep-man/groovy/app/build.gradle

```
@@ -0,0 +1,7 @@
+plugins {
+    id("java-library")
+}
+
+dependencies {
+    implementation(platform(":platform"))
```

```suggestion
    implementation(platform(project(":platform")))
```

I think the previous code would fail during resolution, trying to find a dependency at coordinates group: '', module: 'platform'.

Might be worth adding tests to these snippets to verify something like the `dependencies` task succeeds.

---

The test runs "app:dependencies" for this snippet. I also open these separately as their own project and check them myself.